packages/core/admin/admin/src/content-manager/components/BlocksInput/BlocksToolbar.tsx

```diff
@@ -144,6 +151,38 @@ const ToolbarButton = ({
   );
 };
 
+const handleToggleList = (
+  editor: Editor,
+  currentListEntry: NodeEntry<Ancestor>,
+  format: Block<'list'>['format']
+) => {
+  const [currentList, currentListPath] = currentListEntry;
+
+  if (!Editor.isEditor(currentList) && isListNode(currentList)) {
+    // Format is different, toggle list format
+    if (currentList.format !== format) {
+      Transforms.setNodes(editor, { format }, { at: currentListPath });
+    } else {
+      // Format is same, convert selected list-item to paragraph
+      const [, lastNodePath] = Editor.last(editor, []);
+
+      Transforms.unwrapNodes(editor, {
+        match: (node) => isListNode(node) && ['ordered', 'unordered'].includes(node.format),
```

Comment:
why do we need to check the format? if it is indeed a list I don't think it can be anything else

Reply:
forgot to update